src/io/filesystem.ts

```diff
@@ -140,6 +146,15 @@ export function splitRefStudioPath(filePath: string): string[] {
   return filePath.split(sep);
 }
 
+export function getFileNameAndExtension(filePath: string): { name: string; ext: string } {
+  const fileName = filePath.slice(filePath.lastIndexOf(sep) + 1);
+  const dotIndex = fileName.lastIndexOf('.');
+  if (dotIndex === -1) {
+    return { name: fileName, ext: '' };
+  }
+  return { name: fileName.slice(0, dotIndex), ext: fileName.slice(dotIndex + 1) };
+}
```

Comment:
There is an issue with dot files: for a `.gitignore` file, the function would return `{ name: '', extension: 'gitignore' }`
Even if we don't have dot files for now, I think it would be safer to treat this edge case